dapps/marketplace/src/mutations/SetNetwork.js

```diff
@@ -2,6 +2,6 @@ import gql from 'graphql-tag'
 
 export default gql`
   mutation SetNetwork($network: String!, $customConfig: ConfigInput) {
-    setNetwork(network: $network, customConfig: $customConfig)
```

Comment:
Hrm should this have been removed?

Reply:
No... not at all.  And I'm quite curious how I didn't notice this in testing since my changes depend on this.  I even added support for the `customConfig` param in the backend in this PR...

Thank you for catching this!